catalog-info.yaml

```diff
@@ -150,81 +155,3 @@ spec:
           access_level: "READ_ONLY"
         ingestion-team: {}
         search-productivity-team: {}
-
----
-apiVersion: "backstage.io/v1alpha1"
-kind: "Resource"
-metadata:
-  name: "connectors-nightly-dockers"
-  description: "Connectors Service Nightly Dockers"
-  links:
-    - title: "connectors Nightly Buildkite Jobs"
-      url: "https://buildkite.com/elastic/connectors"
-spec:
-  type: "buildkite-pipeline"
-  owner: "group:ingestion-team"
-  system: "buildkite"
-  implementation:
-    apiVersion: "buildkite.elastic.dev/v1"
-    kind: "Pipeline"
-    metadata:
-      name: "connectors-nightly-dockers"
-      description: "Connectors Service Nightly Dockers"
-      links:
-        - title: "Connector Service Nightly Pipeline"
-          url: "https://buildkite.com/elastic/connectors-nightly-dockers"
-    spec:
-      pipeline_file: ".buildkite/nightly_docker.yml"
-      provider_settings:
-        trigger_mode: "none"
-      repository: "elastic/connectors"
-      schedules:
-        Daily 8_14:
-          branch: '8.14'
-          cronline: '@daily'
-          message: "Builds and pushes daily `8.14` docker images"
-        Daily 8_15:
-          branch: '8.15'
-          cronline: '@daily'
-          message: "Builds and pushes daily `8.15` docker images"
-        Daily main:
-          branch: main
-          cronline: '@daily'
-          message: "Builds and pushes daily `main` docker images"
-      teams:
-        everyone:
-          access_level: "READ_ONLY"
-        ingestion-team: {}
-        search-productivity-team: {}
-
-########
-# Docker image build and publish - manual release
-########
----
-apiVersion: "backstage.io/v1alpha1"
-kind: "Resource"
-metadata:
-  name: "connectors-docker-build-publish"
```

Comment:
Similar question, is this safe to remove, or will it delete this pipeline for all branches if backstage doesn't see it in `main`?

Reply:
I've looked at the following docs:
* https://docs.elastic.dev/ci/catalog/catalog-info-setup
* https://docs.elastic.dev/ci/getting-started-with-buildkite-at-elastic#2-adjust-your-pipeline-settings

I think we have to preserve these `catalog-info.yaml` bits in `main`, while we could remove the unneeded pipelines definitions and scripts from `main`. @acrewdson Do you also see the same?

Reply:
yep, that would also be my guess